src/lib/db/clients/mysql.js

```diff
@@ -190,9 +190,9 @@ export async function getPrimaryKey(conn, database, table) {
 export async function getTableKeys(conn, database, table) {
   const sql = `
     SELECT constraint_name as 'constraint_name', column_name as 'column_name', referenced_table_name as 'referenced_table_name',
-      CASE WHEN (referenced_table_name IS NOT NULL) THEN 'FOREIGN'
-      ELSE constraint_name
-      END as key_type
+      IF(referenced_table_name IS NOT NULL, 'FOREIGN', constraint_name) as key_type,
+      REFERENCED_TABLE_NAME as referenced_table,
+      REFERENCED_COLUMN_NAME as referenced_column
```

Comment:
I think actually this needs a tweak to change the `where` -- we need to remove the primary key from the filter.

line 199 - remove `or constraint_name like '%PRIMARY%'`

Reply:
while developing i was confused why the primary are alsi needed, but i didnt know why so leaved it.
I will remove it :+1:

Reply:
done